core/src/manifest/network.rs

```diff
@@ -11,6 +17,8 @@ pub struct Network {
 
     pub rpc: String,
 
+    pub kind: ProviderType,
```

Comment:
yeah defo not keen on this approach would rather auto work it out by rpc URL IMHO

Reply:
i could think only about 2 options:
1. Deduce type based on url
2. Implicitly specify it
First have the problem that it could fail for no apparent reason (change from using https://1.hypersync.xyz to localproxy/kubernetes service name will break it silently with no way to fix it unless code is changed) 
Second is too verbose

Wdyt about combining approaches -> We deduce from url if no kind provided?

Reply:
yeah ok we can make it default RPC and then not add it to all examples but just have 1 example of how to enable it for hypersync?